.github/README.md

```diff
@@ -3,39 +3,28 @@
 [![Build debug APK](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_debug_apk.yml/badge.svg)](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_debug_apk.yml)
 [![Build release APK](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_release_apk.yml/badge.svg)](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_release_apk.yml)
 [![Discord](https://img.shields.io/discord/803299970169700402?label=server&logo=discord)](https://discord.gg/lawnchair-803299970169700402)
+[![License is Apache 2](https://img.shields.io/github/license/LawnchairLauncher/lawnicons)](./LICENSE)
 
 **Lawnicons** is an icon pack developed by the Lawnchair team.
 Originally an addon for Lawnchair 12 Alpha 5 and above to implement themed icons, it can now be used on many launchers. Some launchers may also support Lawnicons for themed icons.
 
-Lawnicons is best used on Lawnchair 12.1. You can enable themed icons on Lawnchair by going to `Home Settings > General > Icon Style > Themed Icons`, choosing "Home Screen" or "Home Screen and App Drawer".
+Lawnicons is best used on [Lawnchair 12.1](https://github.com/LawnchairLauncher/lawnchair). You can enable themed icons on Lawnchair by going to `Home Settings > General > Icon Style > Themed Icons`, choosing "Home Screen" or "Home Screen and App Drawer".
 
-This project is mostly supported by our community. If you wish to contribute, see the [Contributing section](README.md#Contributing) below.
+This project is mostly supported by our community. If you wish to contribute, check out the Contributing section below.
 
 ## Download
 See the [Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) to download the latest stable build of Lawnicons.
 
-For the development versions with new icons, go to [nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
+For the development versions with new icons, go to [the nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
 
 ## Contributing
 
 Please see the [contributing guide](CONTRIBUTING.md) for information on adding icons or contributing code.
 
-For users familiar with the guidelines, check out [how to use icontool.py](/.github/icontool_guide.md) to speedup the process of adding icons.
+If you know how to create proper icons already, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding icons. Also see the [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613).
```

Comment:
```suggestion
If you're already familiar with the guidelines for creating icons and you created an icon for lawnicons, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding your icons. Otherwise our [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613) is a great start.
```
I think this would be the clearest